hoverxref/extension.py

```diff
@@ -318,6 +318,27 @@ def deprecated_configs_warning(app, exception):
         app.config.hoverxref_api_host = app.config.hoverxref_tooltip_api_host
 
 
+def setup_jquery(app, exception):
+    """
+    Inject jQuery if Sphinx>=6.x
+
+    Staring on Sphinx 6.0, jQuery is not included with it anymore.
+    As this extension depends on jQuery, we are including it when Sphinx>=6.x
+    """
+
+    if sphinx.version_info >= (6, 0, 0):
+        # https://jquery.com/download/#using-jquery-with-a-cdn
+        jquery_cdn_url = "https://code.jquery.com/jquery-3.6.0.min.js"
```

Comment:
I noted similar on our theme, it's better to include dependencies via package dependencies for a few reasons:

https://github.com/readthedocs/sphinx_rtd_theme/pull/1299#discussion_r911189794

Reply:
Also, this dependency can be brought in without webpack, though a longer term goal would be to use webpack, or more likely to remove jQuery entirely.

To copy the jQuery release into the package, install jQuery via `package.json` and copy the JS minified file from `node_modules/` into the package.

Of course the issues you noted with jQuery versions and conflicting versions between extensions is going to be a problem.

Reply:
> To copy the jQuery release into the package, install jQuery via `package.json` and copy the JS minified file from `node_modules/` into the package.

is this better than just grabbing the official minified version from https://jquery.com/download/?

Reply:
Yup. It's much easier to keep up to date and we're using npm to manage these dependencies on our other JS projects.